### DISCUSSION TITLE
Paste from clipboard sometimes will appear extra blankline

### DISCUSSION THREAD

**Discussion by r-xiumu** [wezterm/wezterm#2358] [General]
#### Paste from clipboard sometimes will appear extra blankline

example:
clipboard in windows idea
```
aaa
bbb
ccc
```

pasted to linux
```
aaa

bbb

ccc
```

**Comment by wez** [comment]
Take a look at: https://wezfurlong.org/wezterm/config/lua/config/canonicalize_pasted_newlines.html 
especially the last couple of paragraphs

**Reply by r-xiumu** [reply]
Thanks very much.
